blogo-input/posts/votedelegation.md

```diff
@@ -0,0 +1,105 @@
+# Adding Vote Delegation to Anonymous E-Voting Schemes
+
+*2022-05-06 by [Vincenzo Iovino](#)*
```

Comment:
@vincenzoiovino you can add here a link to your page/twitter/github/url replacing the `#`.